seed/challenges/08-coding-interview-questions-and-take-home-assignments/coding-interview-data-structure-questions.json

```diff
@@ -674,31 +674,29 @@
       "id": "587d8254367417b2b2512c70",
       "title": "Creating and Adding to Sets in ES6",
       "description": [
-        "Now we have worked through ES5, we are going to perform similar actions in ES6. This will be considerably easier. Set is now now a provided data structure like Object, Array, etc. so many of the operations we wrote by hand are now included for us. Let's take a look:",
+        "Now that you have worked through ES5, you are going to perform something similar in ES6. This will be considerably easier. ES6 contains a built-in data structure <code>Set</code> so many of the operations you wrote by hand are now included for you. Let's take a look:",
         "To create a new empty set:",
-        "var set = new Set();",
+        "<code>var set = new Set();</code>",
         "You can create a set with a value:",
-        "var set = new Set(1);",
+        "<code>var set = new Set(1);</code>",
         "You can create a set with an array:",
-        "var set = new Set([1,2,3]);",
-        "Once you have created a set, you can add the values you wish using set.add:",
-        "var set = new Set([1,2,3]);",
-        "set.add([4,5,6]);",
-        "For this exercise, we will return a set with the following values: 1, 2, 3, 'Taco', 'Cat', 'Awesome'",
-        "Test suite (using the assert method)",
-        "assert((function(){var test = checkSet(); var testArr = [...test];  testArr === [ 1, 2, 3, 'Taco', 'Cat', 'Awesome'])}, 'message: Your Set was returned correctly!');"
+        "<code>var set = new Set([1,2,3]);</code>",
+        "Once you have created a set, you can add the values you wish using <code>set.add</code>:",
+        "<blockquote>var set = new Set([1,2,3]);<br>set.add([4,5,6]);</blockquote>",
+        "For this exercise, return a set with the following values: <code>1, 2, 3, 'Taco', 'Cat', 'Awesome'</code>"
       ],
       "challengeSeed": [
         "function checkSet(){",
         "   var set = new Set([1,2,3,3,2,1,2,3,1]);",
```

Comment:
Maybe we want to add a comment on the line before line 690 of this diff, saying something like `// The set constructor removes any duplicates.`

Or perhaps even add it to the description with an updated example. As it's the core feature of Sets, it might be worth repeating - what do you think? :blush: